src/components/schemas/cost_center.yaml

```diff
@@ -60,21 +60,38 @@ cost_center_in:
     is_enabled:
       $ref: ./fields.yaml#/is_enabled
     restricted_spender_user_ids:
-      $ref: ./fields.yaml#/restricted_spender_user_ids
-    org_id:
-      $ref: ./fields.yaml#/org_id
-
-cost_center_restrict_in:
-  type: object
-  required:
-  - id
-  - allow_users
-  additionalProperties: False
-  properties:
-    id:
-      $ref: './fields.yaml#/id_integer'
-    allow_users:
-      $ref: ./fields.yaml#/allow_users
+      type: array
+      nullable: True
+      items:
+        type: string
+        nullable: False
+        maxLength: 15
+        description: |
+          This id is provided by Fyle to identify an object.
+        example: 'sdfd2391'
```

Comment:
change this example

Reply:
+1

Reply:
Fixed this and merging off. Thanks folks